packages/server/src/ChildProcess.ts

```diff
@@ -76,13 +77,15 @@ export class ChildProcess {
             const { startingNodeIds, depthQueue } = getStartingNodes(nonDirectedGraph, endingNodeId)
 
             /*** BFS to traverse from Starting Nodes to Ending Node ***/
+            const chatId = await getChatId(chatflow.id)
```

Comment:
this will not work because child processes dont have access to `DataSource`. You'll have to get the chatid in `startChildProcess()` and pass it in the `IRunChatflowMessageValue` - https://github.com/FlowiseAI/Flowise/blob/main/packages/server/src/index.ts#L449. Then retrieve it here - https://github.com/FlowiseAI/Flowise/blob/main/packages/server/src/ChildProcess.ts#L26